diff -Naur linux-xlnx_orig/arch/arm/boot/dts/picozed-zynq7-board.dtsi linux-xlnx/arch/arm/boot/dts/picozed-zynq7-board.dtsi
--- linux-xlnx_orig/arch/arm/boot/dts/picozed-zynq7-board.dtsi	2015-06-25 16:21:37.926518649 +0100
+++ linux-xlnx/arch/arm/boot/dts/picozed-zynq7-board.dtsi	2015-06-24 13:12:47.435125417 +0100
@@ -57,7 +57,7 @@
 		} ;
 
 		/* Disabled Devices */
-		ps7_sd_0: ps7-sdio@e0100000 { compatible = "invalid"; };
+		//ps7_sd_0: ps7-sdio@e0100000 { compatible = "invalid"; };
 		ps7_uart_0: serial@e0000000 { compatible = "invalid"; };
 		ps7_ethernet_1: ps7-ethernet@e000c000 { compatible = "invalid"; };
 		ps7_i2c_0: ps7-i2c@e0004000 { compatible = "invalid"; };
